src/graph/delta_matrix/delta_matrix.h

```diff
@@ -104,15 +103,10 @@ GrB_Info Delta_Matrix_new
 	bool transpose           // if true, create a transpose of the matrix
 );
 
-// returns transposed matrix of C
-Delta_Matrix Delta_Matrix_getTranspose
-(
-	const Delta_Matrix C
-);
-
 GrB_Matrix Delta_Matrix_M
 (
-	const Delta_Matrix C
+	const Delta_Matrix C,
+	bool transpose
 );
```

Comment:
I think it would be better to keep `Delta_Matrix_M` as it was before this change.
if someone wants `M` transposed, he/she will first call `Delta_Matrix_getTranspose` to get the transposed `DeltaMatrix`  `AT` and then call `Delta_Matrix_M` to get its `M` internal matrix.